GOOD-I-DEER/nodes/webcam-object-detection-node/webcam-html.js

```diff
@@ -94,10 +96,7 @@ module.exports.code = (config) => {
                     const boxes = await detect_objects_on_image(buffer);
                     
                     if(ws.OPEN){
```

Comment:
please move this code to 92 line.
this code will be executed every setInterval callback.